criu/files-reg.c

```diff
@@ -1581,6 +1587,110 @@ static int get_build_id(const int fd, const struct stat *fd_status,
 	return ret;
 }
 
+static inline void calculate_checksum_iterator_init(u64 *iter)
```

Comment:
The name of this function is too long. It is used just in this file, so you can use a shorter name. The code will be more readable.